### PR TITLE
Added read less button when collapsing text

### PR DESCRIPTION
**Pull request by bfheythem**
I made some changes to add a "read less" text, to make the collapse event more intuitive

### CODE REVIEW

File: source/components/ReadMoreReact.jsx [hunk 1]
```diff
@@ -60,8 +62,11 @@ ReadMoreReact.propTypes = {
   ideal: PropTypes.number,
   max: PropTypes.number,
 	readMoreText: PropTypes.string,
+	readLessText: PropTypes.string,
+
 };
 
 ReadMoreReact.defaultProps = {
   readMoreText: "read more",
+  readLessText: "read more",
```
Comment (caseyjkey) [inline source/components/ReadMoreReact.jsx]: should be `read less`

File: README.md [hunk 2]
```diff
@@ -28,13 +28,17 @@ Additional Parameters: You can customize the starting point (min), the ideal len
 | ideal              | 100                        |
 | max                | 200                        |
 | readMoreText       | "read more"                |
+| readLessText       | "read Less"                |
```
Comment (caseyjkey) [inline README.md:31]: should be `read less`

**Review by caseyjkey — APPROVED** [review]

**Comment by caseyjkey** [comment]
@alexandersmanning I looked through this PR it's almost perfect.

**Comment by alexandersmanning** [comment]
I am going to approach this in a different way to provide both functionalities

**Comment by caseyjkey** [comment]
@alexandersmanning react-read-more-read-less has both

**Pull request closed without merging**